### PR TITLE
Upgradle to 5.5.1

### PR DESCRIPTION
**Pull request by wolfs**

### CODE REVIEW

File: src/test/groovy/org/gradle/profiler/ChromeTraceIntegrationTest.groovy [hunk 1]
```diff
@@ -17,7 +17,7 @@ class ChromeTraceIntegrationTest extends AbstractProfilerIntegrationTest {
         new File(outputDir, "${versionUnderTest}-trace.html").isFile()
 
         where:
-        versionUnderTest << supportedGradleVersions
+        versionUnderTest << ["4.7"]
```
Comment (lptr) [inline src/test/groovy/org/gradle/profiler/ChromeTraceIntegrationTest.groovy]: Why not test with all versions as before?

File: subprojects/chrome-trace/src/main/java/org/gradle/trace/GradleTracingPlugin.java [hunk 2]
```diff
@@ -61,8 +61,16 @@ private long getStartTime() {
             try {
                 return buildRequestMetaData.getStartTime();
             } catch (NoSuchMethodError e) {
-                return buildRequestMetaData.getBuildTimeClock().getStartTime();
+                return (long) call(call(buildRequestMetaData, "getBuildTimeClock"), "getStartTime");
             }
         }
     }
+
+    private Object call(Object object, String method) {
```
Comment (lptr) [inline subprojects/chrome-trace/src/main/java/org/gradle/trace/GradleTracingPlugin.java]: Make this `static` and rename it to `invokeGetter`?

File: src/test/groovy/org/gradle/profiler/ChromeTraceIntegrationTest.groovy (hunk 1 above)
Comment (wolfs) [reply src/test/groovy/org/gradle/profiler/ChromeTraceIntegrationTest.groovy]: Good catch, I changed this when testing out something locally. Reverted to testing all versions again.

File: subprojects/chrome-trace/src/main/java/org/gradle/trace/GradleTracingPlugin.java (hunk 2 above)
Comment (wolfs) [reply subprojects/chrome-trace/src/main/java/org/gradle/trace/GradleTracingPlugin.java]: I moved the method to a utility class.

**Comment by wolfs** [comment]
@lptr PTAL!

**Comment by lptr** [comment]
LGTM.

**Pull request merged**